tests/4.5/target_parallel/test_target_parallel.F90

```diff
@@ -0,0 +1,52 @@
+!===--- test_target_parallel.F90 --------------------------------------------===//
+!
+! OpenMP API Version 4.5 Nov 2015
+! 
+! This test checks for the combined construct target and parallel. It
+! creates a parallel region inside of the target devices.
+! 
+!===-------------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_parallel
+   USE iso_fortran_env
+   USE ompvv_lib
+   USE omp_lib
+   implicit none
+
+   OMPVV_TEST_OFFLOADING
+
+   OMPVV_TEST_VERBOSE(target_parallel() .ne. 0)
+
+   OMPVV_REPORT_AND_RETURN()
+
+   CONTAINS
+      INTEGER FUNCTION target_parallel()
+         INTEGER :: errors, thread_id, i
+         INTEGER, DIMENSION(OMPVV_NUM_THREADS_DEVICE) :: num_threads
+         CHARACTER(len=400) :: threadMsg
+         errors = 0
+         thread_id = 0
+
+         !$omp target parallel num_threads(OMPVV_NUM_THREADS_DEVICE) map(from: num_threads(0:OMPVV_NUM_THREADS_DEVICE))
+            thread_id = omp_get_thread_num()
```

Comment:
race on thread_id . better of line 36 is modified to:
num_threads(omp_get_thread_num()) = omp_get_num_threads()

Reply:
*Check .c file as well for same issue